src/tools/css/get-css.js

```diff
@@ -105,8 +109,11 @@ const getCSS = (fondue) => {
 		let cssvardecs = "";
 
 		for (const feature of features) {
-			const indexedFeature = getFeatureIndex(feature);
-			const defaultState = layoutFeatures[indexedFeature].state;
+			const featureIndex = getFeatureIndex(feature);
```

Comment:
Not for this PR but it's probably worthwhile to add some tests for the CSS output. It's a substantial amount of code that can easily break if something in the engine changes.

Reply:
That is true. CSS will be handled shortly, so let's make it more robust in a future PR!